pkg/capture/capture_manager.go

```diff
@@ -405,43 +410,41 @@ func (cm *Manager) rotate(ctx context.Context, writeoutChan chan<- capturetypes.
 		return
 	}
 
-	var rg RunGroup
+	// Iteratively roate all interfaces. Since the rotation results are put on the writeoutChan for
+	// writeout by the DBWriter (which is sequential and certainly slower than the actual in-memory rotation)
+	// there is no significant benefit from running the rotations in parallel, thus allowing us to minimize
+	// congestion _and_ use a single shared local memory buffer
```

Comment:
See comment. Important: The whole concept is still concurrency-safe (I tested it, the memory pool ensure that no race conditions can occur), I just _choose_ to not do it in parallel because there's no benefit (on the contrary). External status calls are still done in parallel (however in practice are run one by one since the default number of buffers is one). IMHO this should be the optimal use, but it still allows us to pimp for edge cases (we have to test if maybe calls to `Status()` for all interfaces take too long if many interfaces are present).